### PR TITLE
feat: pgbouncer container support

### PR DESCRIPTION
**Pull request by seveibar**
usage

image

### CODE REVIEW

File: src/public-types.ts [hunk 1]
```diff
@@ -32,6 +37,17 @@ export interface GetTestPostgresDatabaseFactoryOptions<
       mode?: BindMode
     }[]
   }
+
+  /**
+   * Pgbouncer container settings, disabled by default. Container settings are
```
Comment (codetheweb) [inline src/public-types.ts:42]: what container settings are inherited?

File: package-lock.json
Comment (codetheweb) [inline package-lock.json:1]: should remove this, migrating to npm can be a separate PR

**Review by codetheweb — APPROVED** [review]
awesome

File: package-lock.json
Comment (seveibar) [reply package-lock.json:1]: woops thank

File: src/public-types.ts (hunk 1 above)
Comment (seveibar) [reply src/public-types.ts:42]: i guess i don't right now, will remove

Comment (seveibar) [reply src/public-types.ts]: c8d7d82

File: package-lock.json
Comment (seveibar) [reply package-lock.json:1]: c8d7d82
